src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+import duke.exception.IllegalCommandException;
+
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public String getInitCommand() {
+        return initCommand;
+    }
+
+    public void setInitCommand(String initCommand) {
+        this.initCommand = initCommand;
+    }
+    public String getSaveString(){
+        return initCommand+" "+(isDone() ? "1":"0")+System.lineSeparator();
+    }
+
+    private String initCommand;
+    public Task(){
+        this.isDone=false;
+        this.description="";
+    }
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+
+        return (isDone() ? "X" : " "); // mark done task with X
+    }
+
+    //...
```

Comment:
Try to remove comments that do not contain related information. If needed GitHub has version tracking for you to retrieve the comments.